cvat/apps/engine/management/commands/syncperiodicjobs.py

```diff
@@ -71,6 +71,8 @@ def handle(self, *args, **options):
                     cron_string=job_definition['cron_string'],
                     func=job_definition['func'],
                     id=job_id,
+                    args=job_definition.get('args'),
+                    kwargs=job_definition.get('kwargs'),
```

Comment:
You need to update the comparison logic to make sure that the job is recreated if its args are changed.